content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -10,98 +10,94 @@ In Odoo, this process can be conducted manually— by warehouse employees physic
-columns, respectively.
-
-.. image:: using_inventory_valuation/inventory-valuation-entry.png
-   :align: center
-   :alt: Accounting entry for the inventory valuation of 10 tables.
-
 Deliver a product
 -----------------
 
 In the same logic, when a table is shipped to a customer and leaves the warehouse, the stock
 valuation decreases. The :guilabel:`Valuation` smart button on the :abbr:`DO (Delivery Order)`,
-likewise, displays the stock valuation record as it does on a :abbr:`PO (Purchase Order)`
+likewise, displays the stock valuation record as it does on a :abbr:`PO (Purchase Order)`.
 
 .. image:: using_inventory_valuation/decreased-stock-valuation.png
    :align: center
    :alt: Decreased stock valuation after a product is shipped.
 
-The inventory valuation report
-------------------------------
+.. _inventory/reporting/valuation-report:
+
+Inventory valuation report
+==========================
 
-To view the current value of all products in the warehouse, go to :menuselection:`Inventory app -->
-Reporting --> Inventory Valuation`. The records in the table are organized by product, and selecting
-a product's drop-down menu displays detailed records with the :guilabel:`Date`,
-:guilabel:`Quantity`, and :guilabel:`Total Value` of the inventory.
+To view the current value of all products in the warehouse, first turn on :ref:`Developer mode
+<developer-mode>` and navigate to :menuselection:`Inventory app --> Reporting --> Valuation`. The
+page displays detailed records of products with the :guilabel:`Date`, :guilabel:`Quantity`,
+:guilabel:`Unit Value`, and :guilabel:`Total Value` of the inventory.
+
+.. important::
+   :ref:`Developer mode <developer-mode>` **must** be enabled to see the :guilabel:`Valuation`
+   option under :guilabel:`Reporting`.
 
 .. image:: using_inventory_valuation/inventory-valuation-products.png
    :align: center
    :alt: Inventory valuation report showing multiple products.
 
-.. note::
-   The :guilabel:`Inventory At Date` button, located in the top-left corner of the :guilabel:`Stock
-   Valuation` dashboard, shows the inventory valuation of products available during a prior
-   specified date.
+The :guilabel:`Valuation At Date` button, located in the top-left corner of the :guilabel:`Stock
+Valuation` page, reveals a pop-up window. In this pop-up, the inventory valuation of products
+available during a prior specified date can be seen and selected.
+
+.. tip::
+   View a detailed record of a product's inventory value, stock move, and on-hand stock by selecting
+   the teal :guilabel:`( ➡️ ) arrow` button to the right of the :guilabel:`Reference` column value.
 
 Update product unit price
-~~~~~~~~~~~~~~~~~~~~~~~~~
+-------------------------
 
 For any company: lead times, supply chain failures, and other risk factors can contribute to
 invisible costs. Although Odoo attempts to accurately represent the stock value, *manual valuation*
```

Comment:
Same with the field values here below on lines 108-109 - if they are from a dropdown please use :guilabels:, otherwise user input use keep it as literal